### DISCUSSION TITLE
[Feature] Looping .webm/.webp option

### DISCUSSION THREAD

**Discussion by nickXIII** [immich-app/immich#3055] [Feature Request]
### The feature

The ability to set an image, album, or set of uploads to automatically loop .webm files would be greatly appreciated. Webm files save a considerable amount of space over traditional GIF format animations.

From my understanding this is a function of how the site displays content, so the simplest option would seem to be a toggle to automatically loop all webm format files, but ideally a per album toggle would allow greater freedom on the user's end.

GIF example image - 1,201 KB
![gif-example](https://github.com/immich-app/immich/assets/33738783/7db08a42-0b9b-4263-b0d5-de0e5a335b9e)

WEBM example image - 187 KB
[webm-example.webm](https://github.com/immich-app/immich/assets/33738783/496944d7-422f-4d22-a3a6-256ea395762c)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3289
